.env-example

```diff
@@ -0,0 +1,5 @@
+SKIP_PREFLIGHT_CHECK=true
+
+REACT_APP_VERSION=$npm_package_version
+
```

Comment:
Why we need those ?

Reply:
SKIP_PREFLIGHT_CHECK - it helps to correct errors.  I used i18nexus because you can choose version there  the default version is the latest

Reply:
It works without it anyway, so I delete it